python/dgl/distributed/dist_graph.py

```diff
@@ -169,7 +170,7 @@ def process_request(self, server_state):
             gpb = server_state.partition_book
             # Initialize the edge attribute.
             num_edges = g.total_num_edges
-            attr_data = torch.zeros(num_edges, dtype=data_type)
+            attr_data = torch.full((num_edges,), self._padding, dtype=data_type)
```

Comment:
what if we simply replace `torch.zeros()` with `torch.ones()` without adding `padding` argument?